packages/components/src/Checkbox/Checkbox.js

```diff
@@ -52,7 +52,7 @@ Checkbox.defaultProps = {
 
 Checkbox.propTypes = {
 	id: PropTypes.string.isRequired,
-	label: PropTypes.string,
+	label: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
```

Comment:
That's not just using RTL and changing lifecycles, that's an large API change 🚓

Reply:
Definition of node:
/ Anything that can be rendered: numbers, strings, elements or an array so you can just replace it by node

Reply:
The propTypes were not correct, the component was already used with components. This is to remove the proptypes warnings

Reply:
Merged your suggestion JM